rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/jobs/JobRequest.groovy

```diff
@@ -26,5 +26,17 @@ import groovy.transform.Immutable
 @CompileStatic
 class JobRequest {
   List<String> tokenizedCommand
+  List<String> maskedPackerParameters = []
+
+  List<String> getMaskedTokenizedCommand() {
+    return tokenizedCommand.collect {
+      String masked = it
+      String key = it.split('=')[0]
+      if (key && maskedPackerParameters.contains(key)) {
```

Comment:
Could use `in` operator instead of `contains()`, but I think it reads fine as-is.